packages/gafl-webapp-service/src/pages/summary/licence-summary/__tests__/route.spec.js

```diff
@@ -50,288 +39,250 @@ jest.mock('../../../../processors/mapping-constants.js', () => ({
     none: 'Not Proof'
   }
 }))
+jest.mock('../../find-permit.js', () => jest.fn())
 
-describe('licence-summary > route', () => {
-  beforeEach(jest.clearAllMocks)
-
-  describe('getFromSummary', () => {
-    it('should return licence-summary, if it is a renewal', async () => {
-      const result = await getFromSummary(undefined, true)
-      expect(result).toBe(LICENCE_SUMMARY_SEEN)
+const getMockRequest = ({ currentPermission = getMockPermission(), statusCache = {}, statusCacheSet = () => {} } = {}) => ({
+  cache: () => ({
+    helpers: {
+      status: {
+        getCurrentPermission: () => statusCache,
```

Comment:
should be async or not?

Reply:
Yes, good spot (it doesn't seem to matter with tests, but it makes the intent clearer if we match the mock to the actual behaviour